src/screens/CreatePartnership/index.tsx

```diff
@@ -1,28 +1,48 @@
 import { Button, Header, Text } from "@components";
 import { PartnershipForm } from "@screens/PartnershipForm";
-import { useState } from "react";
-import { Container, ButtonView, SearchView } from "./styles";
-
+import { useEffect, useState } from "react";
+import { View } from "react-native";
+import { Container, ButtonView, SearchView, PartnerView } from "./styles";
 
 export function CreatePartnership() {
+  const [visibleModal, setVisibleModal] = useState(false);
+  const [data, setData] = useState([]);
 
-    const [visibleModal, setVisibleModal] = useState(false);
-    return (
-        <Container>
-            <Header isHero={true} />
+  useEffect(() => {
+    fetch("https://owlpartners.onrender.com/partners", {})
```

Comment:
É preciso criar uma classe que realizará essas requisições, com a minha implementação (que está em review) você pode se basear no user.request para criar a classe de PartnersRequest

Reply:
Concordo com o Thagão. Criar a classe que agrupa as requisições e utilizar o axios, além de criar um enum com os endpoints que vc tá usando para padronização e futura manutenção (se for preciso)